lib/screens/activities/calmingcliffintro.dart

```diff
@@ -0,0 +1,198 @@
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                FadeTransition(
+                  opacity: _fadeAnimation,
+                  child: RotationTransition(
+                    turns: _leftMountainController,
+                    child: const Icon(
+                      Icons.landscape,
+                      color: Colors.white,
+                      size: 24,
+                    ),
+                  ),
+                ),
+                const SizedBox(width: 20),
+                AnimatedScale(
+                  scale: _isMountainScaled ? 1.2 : 0.8,
+                  duration: const Duration(seconds: 1),
+                  curve: Curves.easeInOut,
+                  child: const Icon(
+                    Icons.terrain,
+                    color: Colors.white,
+                    size: 80,
+                  ),
+                ),
+                const SizedBox(width: 20),
+                FadeTransition(
+                  opacity: _fadeAnimation,
+                  child: RotationTransition(
+                    turns: _rightMountainController,
+                    child: const Icon(
+                      Icons.landscape,
+                      color: Colors.white,
+                      size: 24,
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            const SizedBox(height: 50),
+            AnimatedOpacity(
+              opacity: _isTextVisible ? 1.0 : 0.0,
+              duration: const Duration(seconds: 2),
+              curve: Curves.easeInOut,
+              child: Card(
+                color: Colors.orangeAccent,
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(15),
+                ),
+                child: const Padding(
+                  padding: EdgeInsets.all(20.0),
+                  child: Text(
+                    'Welcome to Calming Cliffs',
+                    style: TextStyle(
+                      fontSize: 24,
+                      fontWeight: FontWeight.bold,
+                      color: Colors.white,
+                    ),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+              ),
+            ),
+            const SizedBox(height: 30),
+            ElevatedButton(
+              style: ElevatedButton.styleFrom(
+                backgroundColor: Colors.white,
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(15),
+                ),
+                foregroundColor: Colors.deepOrangeAccent,
+              ),
+              onPressed: () {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(
+                      builder: (context) => const CalmingCliffsActivity()),
+                ).then((value) {
+                  setActivityCompleted(Future.value(value));

```

Comment:
This is receiving null. Not sure if you're still working on that, but you might be forgetting to pass a value with `Navigator.pop()` in the `CalmingCliffsActivity()`.

Reply:
https://github.com/Kenneth-W-Chen/mindfulness-app/pull/26/commits/99a3c26d1480c9d84845c830e6a27e2cce627bbb